gradle/versions.gradle

```diff
@@ -13,5 +13,38 @@
 
 dependencyManagement {
   dependencies {
+    dependency 'com.fasterxml.jackson.core:jackson-databind:2.10.1'
+
+    dependency 'com.google.errorprone:error_prone_annotation:2.3.4'
```

Comment:
did you want all of this coming in (because we probably don't need all of it - eg auto-service

Reply:
cleaned up, should only have what is needed now